config.go

```diff
@@ -1,46 +1,61 @@
 package main
 
 import (
-	"github.com/devangel/config"
 	"os"
 	"path/filepath"
-	"strings"
-	//"fmt"
+
+	"github.com/devangel/config"
 )
 
 var Config = config.NewConfig("force")
 
-func GetSourceDir() (src string, err error) {
-	// Last element is default
-	var sourceDirs = []string{
-		"src",
-		"metadata",
+var sourceDirs = []string{
+	"src",
+	"metadata",
+}
+
+// IsSourceDir returns a boolean indicating that dir is actually a Salesforce
+// source directory.
+func IsSourceDir(dir string) bool {
```

Comment:
I think the problem with this approach is if you have both a metadata and src directory. Also, package.xml can exist in a "Package" folder.